lib/perl5/YAML/Matrix.pm

```diff
@@ -23,9 +22,8 @@ sub minimal_events_for_framework {
         $args{no_explicit_doc} = 1;
         $args{no_quoting_style} = 1;
     }
-    elsif ($type eq 'rapid') {
-        $args{no_quoting_style} = 1;
-        $args{no_flow_indicator} = 1;
+    elsif ($type eq 'rapidyaml') {
```

Comment:
@perlpunk I'm confused by what should go in here. is `$type` really just `'rapid'`? where does that get set?

Reply:
anyway, this is not doing anything ATM, neither with `'rapid'` nor with `'rapidyaml`'.